fetch_napari_data.py

```diff
@@ -235,9 +255,35 @@ def process_plugin(plugin):
             else {}
         )
         manifest_info = fetch_manifest(plugin_normalized_name)
+        # need pypi info to get the initial and latest release date
+        pypi_info = fetch(urljoin(API_PYPI_BASE_URL, plugin_normalized_name))
 
         if conda_info:
-            flatten_and_merge(plugin_data, conda_info)
+            # we only want a limited set of conda info
+            conda_info = {
+                "conda_name": conda_info["name"],
+                "conda_html_url": conda_info["html_url"],
+                # TODO: this should come from project_url not conda info
+                "home": conda_info["home"],
+            }
+            plugin_data.update(conda_info)
+
+        if pypi_info:
+            # releases are sorted in descending order
+            plugin_first_release = plugin_data["pypi_versions"][-1]
+            plugin_latest_release = plugin_data["pypi_versions"][0]
```

Comment:
minor Q: do you know whether this date includes pre-releases? If so, you probably want to only grab the latest *final* version...